app/src/pages/ODD/ProtocolSetup/index.tsx

```diff
@@ -185,7 +189,20 @@ function PrepareToRun({
     protocolRecord?.data.metadata.protocolName ??
     protocolRecord?.data.files[0].name ??
     ''
-
+  const robotType = useRobotType(robotName)
+  const baseParams = {
+    source: 'runRecord' as const,
+    robotType: robotType,
+  }
+  const { reportCameraEnablementSettings, reportPhotoAccessUsage } =
+    useCameraAnalytics(baseParams)
```

Comment:
```suggestion
    useCameraAnalytics({source: SOURCE_RUN_RECORD, robotType})
```
Nit, I would do this in-line. Just a tad cleaner.